integration-tests/features/steps/common.py

```diff
@@ -325,8 +325,8 @@ def send_manifest_to_stack_analysis(context, manifest, name, endpoint, use_token
 
 
 def stack_analysis_endpoint(context, version):
-    endpoint = {"1": "/api/v1/stack-analyses/",
-                "2": "/api/v1/stack-analyses-v2/"}.get(version)
+    endpoint = {"1": "/api/v1/stack-analyses-v1/",
```

Comment:
@tisnik unfortunately, we won't have any endpoint /stack-analyses-v1. There is one and only one endpoint /stack-analyses.

Reply:
oh ok, thanks for info. I just followed the msg 'changing /stack-analyses-v2 to /stack-analyses and /stack-analyses to /stack-analyses-v1'. Anyway the "1" version is not used in any test AFAIK.